src/api/app/controllers/label_templates_controller.rb

```diff
@@ -0,0 +1,59 @@
+class LabelTemplatesController < ApplicationController
+  before_action :set_label_template, only: %i[update destroy]
+
+  after_action :verify_authorized # raise an exception if authorize has not yet been called.
+
+  # GET /label_templates
+  def index
+    authorize LabelTemplateGlobal
+    @label_templates = LabelTemplateGlobal.all
+
+    render 'label_templates/index', formats: [:xml]
+  end
+
+  # POST /label_templates
+  def create
+    authorize LabelTemplateGlobal
+    @label_template = LabelTemplateGlobal.new(label_template_params)
```

Comment:
```suggestion
    @label_template = LabelTemplateGlobal.new(label_template_params)
    authorize @label_template
```

Reply:
I would prefer to check if a user is allowed to create a global label template before reading the body of the create request. Why do you suggest the other way around?

Reply:
Because it's the "normal" way to authorize the object you are trying to create and not the class. That you can call it only with class and it's still works is a implementation detail of `LabelTemplateGlobalPolicy`.

Reply:
a.k.a. "it looks weird"

Reply:
Suggestion applied.

However, inside [this section](https://github.com/varvet/pundit?tab=readme-ov-file#policies) of the main page of the Pundit project:

_"If you don't have an instance for the first argument to authorize, then you can pass the class."_.

That use case is for checking the permission of a list of elements, though.